### DISCUSSION TITLE
Find barcode on scanned page

### DISCUSSION THREAD

**Discussion by dontbesoso** [cyanfish/naps2#54] [Development]
Hey!

I'm currently testing naps2 to use in our office.

Currently we're scanning documents to put them in DMS.

First page of every document has a barcode sticker with document ID from DMS (CODE 128).

Is there any option to give NAPS a feature which can read a barcode from page and maybe sugest this id as a filename in "Save as pdf" command?

**Comment by tony-xlh** [comment]
There is the [Dynamic .NET TWAIN SDK](https://www.dynamsoft.com/dotnet-twain/overview/) and the [Dynamsoft Barcode Reader SDK](https://www.dynamsoft.com/barcode-reader/sdk-desktop-server/) which suit your needs.
